src/theory/quantifiers/ceg_t_instantiator.cpp

```diff
@@ -385,8 +394,11 @@ bool ArithInstantiator::processAssertion( CegInstantiator * ci, SolvedForm& sf,
   return false;
 }
 
-bool ArithInstantiator::processAssertions( CegInstantiator * ci, SolvedForm& sf, Node pv, std::vector< Node >& lits, unsigned effort ) {
- if( options::cbqiModel() ){
+bool ArithInstantiator::processAssertions(CegInstantiator* ci, SolvedForm& sf,
+                                          Node pv, std::vector<Node>& lits,
+                                          std::vector<Node>& alits,
```

Comment:
`alits` is not used at all here?

Reply:
This is a virtual function, so it follows the interface.